database/user.go

```diff
@@ -70,14 +68,21 @@ type (
 		ID              primitive.ObjectID `bson:"_id,omitempty" json:"-"`
 		Sub             string             `bson:"sub" json:"sub"`
 		Tier            int                `bson:"tier" json:"tier"`
-		StorageUsed     int64              `bson:"storage_used" json:"storageUsed"`
 		SubscribedUntil time.Time          `bson:"subscribed_until" json:"subscribedUntil"`
 	}
-	// UserDetails builds on top of User and provides additional information,
-	// typically more expensive to fetch than a single DB query.
-	UserDetails struct {
-		User
-		BandwidthUsed int64 `bson:"-" json:"bandwidthUsed"`
+	// UserStats contains statistical information about the user.
+	UserStats struct {
+		StorageUsed        int64 `bson:"-" json:"storageUsed"`
+		NumRegReads        int64 `bson:"-" json:"numRegReads"`
```

Comment:
what's the deal with the `bson:"-"`?

Reply:
I don't really need them. I just got used to the duality of bson/json and tunnel-visioned on this one. Removed.